core-api/src/test/java/com/optimizely/ab/event/internal/EventBuilderV2Test.java

```diff
@@ -90,6 +90,7 @@ public void createImpressionEvent() throws Exception {
         assertThat(impression.getVisitorId(), is(userId));
         assertThat((double)impression.getTimestamp(), closeTo((double)System.currentTimeMillis(), 60.0));
         assertFalse(impression.getIsGlobalHoldback());
+        assertThat(impression.getAnonymizeIP(), is(projectConfig.getAnonymizeIP()));
```

Comment:
let's make this comparison explicit. assertFalse in this case since it's a v2 datafile. We should also pass in v2 datafile to the eventbuilder as well as a v3 datafile since for v2, `anonymizeI`P should be `false` by default, while v3 will vary